Update networking widgets so they show a Raspberry Pi icon and arrows pointing towards/out of it.

Williangalvani opened the pull request:
![image](https://github.com/user-attachments/assets/374bae08-a20a-4297-8d40-433600d4e23f)

Also adds a wifi option as a bonus

(I bet sourcery will complain of the v-html)

Williangalvani commented:
latest iteration
![image](https://github.com/user-attachments/assets/af01436e-87f8-46c9-a2d3-b146d7217722)

ES-Alexander commented on core/frontend/src/widgets/Networking.vue:
```diff
@@ -1,26 +1,51 @@
 
 export default Vue.extend({
-  name: 'CpuWidget',
+  name: 'ETh0Widget',
```
```suggestion
  name: 'Eth0Widget',
```
Agreed with Sourcery on this one - probably just a typo to start with.

ES-Alexander reviewed (changes requested):
Did you forget to push? The new RX/TX arrows aren't included yet.

Also, could we add a widget for USB0 as well? And possibly UAP0 (if it's not included in the Wifi one)?

ES-Alexander commented:
Arrow colours don't seem to be working properly in dark mode:

https://github.com/user-attachments/assets/c450a75d-13d4-4128-8b79-9af8b97e2727

The failsafe icon outlines look fine though, so not sure what's going on.

---

Also, were you intending to add those extra widgets, or save that for a different PR?

Williangalvani commented:
@ES-Alexander I updated the PR. the issues with the arrows was likely that I lost information as I had to change them in inkscape.

we are now also dynamically adding entries for the other networks, and I fixed formatBandwidth which was off by 8!
![image](https://github.com/user-attachments/assets/231b11f4-6b07-46d9-aa6c-01a314c6ac33)
